src/components/templates/ThreeColumnsWithTopBar/ThreeColumnsWithTopBar.tsx

```diff
@@ -6,15 +6,22 @@ export const ThreeColumnsWithTopBar = ({
     children: [React.ReactNode, React.ReactNode, React.ReactNode];
 }) => {
     return (
-        <div className='grid h-fit grid-cols-1 place-items-stretch gap-x-3 gap-y-4 tablet:grid-cols-2 laptop:grid-cols-4'>
-            <div className='tablet:col-span-2 laptop:col-span-4'>{topBar}</div>
-            <div className='tablet:col-span-2 laptop:col-span-1'>
+        <div
+            className='grid h-fit grid-cols-1 place-items-stretch gap-x-3
+            tablet:grid-cols-2 laptop:flex laptop:grid-cols-12 laptop:flex-wrap laptop:gap-y-4'
+        >
+            <div className='order-1 tablet:col-span-2 laptop:w-full'>
+                {topBar}
+            </div>
+            <div className='order-2 mb-4 block tablet:col-span-2 laptop:hidden'>
                 {children[0]}
             </div>
-            <div className='hidden laptop:col-span-1 laptop:block'>
+            <div className='order-4 grid grid-cols-12 gap-3 tablet:col-span-2 laptop:flex laptop:w-1/2 laptop:grow laptop:flex-wrap'>
                 {children[1]}
             </div>
-            <div className='laptop:col-span-2'>{children[2]}</div>
+            <div className='order-3 tablet:col-span-2 laptop:order-4 laptop:w-[25%] laptop:min-w-[339px] laptop:max-w-[447px]'>
```

Comment:
Size of Advanced Lending Order Card: minimum 339px, maximum 447px, and 25% of total width in between the two values